recipes-kernel/linux/files/bcm2712-raspberrypi5-usb.dts

```diff
@@ -0,0 +1,262 @@
+/dts-v1/;
+
+/ {
+	#address-cells = <0x2>;
+	#size-cells = <0x2>;
+
+	gic: gic {
+		#interrupt-cells = <0x3>;
+		interrupt-controller;
+	};
+
+	passthrough {
+		compatible = "raspberrypi,5-model-b", "brcm,bcm2712", "simple-bus";
+		ranges;
+		#address-cells = <0x2>;
+		#size-cells = <0x2>;
+
+		clk_xosc: clk_xosc {
+			compatible = "fixed-clock";
+			#clock-cells = <0>;
+			clock-output-names = "xosc";
+			clock-frequency = <50000000>;
+		};
+
+		macb_pclk: macb_pclk {
+			compatible = "fixed-clock";
+			#clock-cells = <0>;
+			clock-output-names = "pclk";
+			clock-frequency = <200000000>;
+		};
+
+		macb_hclk: macb_hclk {
+			compatible = "fixed-clock";
+			#clock-cells = <0>;
+			clock-output-names = "hclk";
+			clock-frequency = <200000000>;
+		};
+
+		mip0: msi-controller@1000130000 {
+			compatible = "brcm,bcm2712-mip-intc";
+			reg = <0x10 0x00130000 0x0 0xc0>;
+			interrupt-parent = <&gic>;
```

Comment:
I am interested how it is working, given that xen provides special `phandle` for gic